src/app/page/UserProfile/ProfileImage.jsx

```diff
@@ -69,7 +69,7 @@ const ProfileImage = ({ classes, profile, setProfile }) => {
   ) : (
     <Grid item>
       <Avatar className={classes.profileImage} src={profilePhoto} alt={displayName} />
-      <Link style={{ textDecoration: "none" }} onClick={setEdit}>
+      <Link to="" style={{ textDecoration: "none" }} onClick={setEdit}>
```

Comment:
Link's an element intended for router navigation. We're gonna come across some issues not using this as intended either around some required Link props or accessibility issues with this having a callback rather than initiating navigation.

Reply:
Yup, you can use the default `Button` from `material-ui/core` instead of the one from components.
See the docs to make it look like you want https://material-ui.com/components/buttons/#button

Reply:
Addressed in e552888